mne/viz/montage.py

```diff
@@ -1,5 +1,8 @@
 """Functions to plot EEG sensor montages or digitizer montages."""
-from ..utils import check_version
+import numpy as np
+from scipy.spatial.distance import cdist
```

Comment:
this import will need to be nested (to avoid `import mne` slowdown we nest most of the `scipy` imports)